app/controllers/concerns/api/v2/lookup_keys_common_controller.rb

```diff
@@ -144,4 +122,11 @@ def return_if_smart_mismatch
       not_found "#{obj} not found by id '#{id}'"
     end
   end
+
+  private
+
+  def model_not_found(model_name)
+    not_found(:error => { :message => (_("#{model_name.capitalize} with id '%{id}' was not found") %
```

Comment:
I understand I should move the `model_name.capitalize` to the % hash. Not sure about the 2nd comment, maybe I should use `model.class.to_s` to make sure it's part of one of the elements in `locale/model_attributes`?